09_proxmox_fstrim_cloudinit_ansible_lvm_08-sep-2022/simple-https/README.md

```diff
@@ -26,30 +26,28 @@ example-com.conf  make-cert.sh  simple-server.py
 ### Creation ###
 
 1. Run make-cert.sh first in order to generate certificate.
+```
+./make-cert.sh
+```
 
 2. Check if the python script can be executed without errors:
 ```
-cd ./simple-https
 python3 simple-server.py
 ```
  
 3. In order to start it from the service it's necessary to create it:
 ```
-cd /etc/systemd/system/
-sudo nano simple-https.service
-```
-
-The service code is below:
-```
+sudo cat <<EOF > /etc/systemd/system/simple-https.service
 [Unit]
 Description=Simple HTTPS Server
 [Service]
-User=tatyana
-WorkingDirectory=/home/tatyana/
+User=$USER
+WorkingDirectory=$HOME
 Type=simple
-ExecStart=/usr/bin/python3 /home/tatyana/simple-https/simple-server.py
+ExecStart=/usr/bin/python3 $HOME/simple-https/simple-server.py
```

Comment:
Are you sure this path will work? The problem is that services are launched from `root` user, thus, `$USER` in this case equals to `root` and `$HOME` to `/root`, that is not connected to the user who creates the service. Please double check this and if necessary adjust both instructions on where to put the files with the code and/or the service code. In any case I believe it is not so good to reference environment variables in such files. Possible it would be better to substitute their values before creating the file. May be I'm wrong, but please double check that the final file `simple-https.service` doesn't reference to some environment variables.